src/test/java/app/komunumo/data/demo/DemoDataCreatorJsonIT.java

```diff
@@ -43,6 +43,8 @@ void testDemoDataImporterWithJson() {
         page.waitForSelector("h1:has-text('Komunumo Test')");
         captureScreenshot("home-page-with-demo-data");
 
+        page.waitForSelector("vaadin-card");
```

Comment:
This change is okay, but it would be better in a separate PR because it does not belong to #288.